documentation/source/release_notes.rst

```diff
@@ -4,6 +4,972 @@ Release Notes
 
 All releases are available from the `GitHub Releases Page <https://github.com/cocotb/cocotb/releases>`_.
 
+.. towncrier release notes start
+
+Cocotb 1.4.0.dev0 (2020-02-07)
```

Comment:
Of course, this doesn't belong in this PR, but my VPN is down so I have to fix it later.